packages/react-core/src/components/DescriptionList/examples/DescriptionList.md

```diff
@@ -120,6 +120,40 @@ import FlagIcon from '@patternfly/react-icons/dist/esm/icons/flag-icon';
 ```ts file='./DescriptionListDefaultInlineGrid.tsx'
 ```
 
+### Display variant
+
+```ts file='./DescriptionListDisplay.tsx'
+```
+
+## Card variants
+
+Cards can be used as description list group wrappers. Using cards in this way applies the card body padding directly to the card and enables the use of card modifiers within description list groups.
```

Comment:
I think it'd be worth linking to our card component here, to make it easier for users to navigate to it if they want:

```suggestion
A [card component](/components/card) can be used as description list group wrappers. Using cards in this way applies the card body padding directly to the card and enables the use of card modifiers within description list groups.
```